cgr/cgr.py

```diff
@@ -51,19 +51,25 @@ async def update_ratings_chameleon(self, ctx, players, winner):
         await self.update_ratings_helper(ctx, players, r2, 0.4, "chameleon")
 
     async def update_ratings_whosaidit(self, ctx, players, winners):
+        await self.update_ratings_who_helper(ctx, players, winners, "whosaidit")
+
+    async def update_ratings_whopostedit(self, ctx, players, winners):
+        await self.update_ratings_who_helper(ctx, players, winners, "whopostedit")
+
+    async def update_ratings_who_helper(self, ctx, players, winners, game_name):
         if len(players) == 1:
             return
         elif winners:
             players.remove(winners[0])
-            loser_rating = await self.get_average_rating(players, "whosaidit")
-            winner_rating = await self.get_average_rating([winners[0]], "whosaidit")
+            loser_rating = await self.get_average_rating(players, game_name)
+            winner_rating = await self.get_average_rating([winners[0]], game_name)
             r2 = math.pow(10, loser_rating/400)
-            await self.update_ratings_helper(ctx, [winners[0]], r2, 1.0 + (0.2 * len(players)), "whosaidit")
+            await self.update_ratings_helper(ctx, [winners[0]], r2, 1.0 + (0.2 * len(players)), game_name)
             r2 = math.pow(10, winner_rating/400)
-            await self.update_ratings_helper(ctx, winners[1:], r2, 0.4, "whosaidit")
-            await self.update_ratings_helper(ctx, [p for p in players if p not in winners], r2, 0, "whosaidit")
+            self.update_ratings_helper(ctx, winners[1:], r2, 0.4, game_name)
+            self.update_ratings_helper(ctx, [p for p in players if p not in winners], r2, 0, game_name)
```

Comment:
Is this missing `await`?

```suggestion
            await self.update_ratings_helper(ctx, winners[1:], r2, 0.4, game_name)
            await self.update_ratings_helper(ctx, [p for p in players if p not in winners], r2, 0, game_name)
```